### ISSUE TITLE
Swipe to close animation does not go perfectly vertical

### ISSUE THREAD

**Issue by likwidfuzion** [rpetrich/ProSwitcher#265]
When you swipe to close, the animation tends to veer off to the left.

**Comment by grp** [comment]
That's true, and it does happen: but only after you release and it shoots off itself. It'd be very hard to fix, but I'll leave it here for reference.
